File: fb_js/fizzbuzz.js
function fizzbuzz() {
  let output = new Array;
  for (let i = 1; i <= 100; i++) {
    let result = ""
    if (i % 3 == 0) result += "Fizz"
    if (i % 5 == 0) result += "Buzz"
    if (result == "") result = i.toString()
    output.push(result)
  }
  return output
}

function printArray(array) {
  let output = "";
  for (let i = 0; i < array.length; i++) {
    output += array[i] + " ";
  }
  console.log(output);
}

printArray(fizzbuzz())

File: fb_go/fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func fizzbuzz() (output [100]string) {
	for i := 1; i < 101; i++ {
		var result string
		if i%3 == 0 {
			result += "Fizz"
		}
		if i%5 == 0 {
			result += "Buzz"
		}
		if result == "" {
			result = strconv.Itoa(i)
		}
		output[i-1] = result
	}
	return output
}

func printArray(array [100]string) {
	for i := 0; i < len(array); i++ {
		fmt.Printf("%v ", array[i])
	}
}

func main() {
	printArray(fizzbuzz())
}
